src/js/pre-need/config/form.js

```diff
@@ -84,32 +88,35 @@ const formConfig = {
-                  },
-                  ssn: ssnUI,
+                  name: fullMaidenNameUI,
+                  ssn: ssnDashesUI,
                   dateOfBirth: currentOrPastDateUI('Date of birth'),
                   relationshipToVet: {
-                    type: {
-                      'ui:title': 'Relationship to Servicemember',
-                      'ui:widget': 'radio',
-                      'ui:options': {
-                        labels: {
-                          1: 'I am the Servicemember',
-                          2: 'Spouse or surviving spouse',
-                          3: 'Unmarried adult child',
-                        }
-                      }
-                    },
-                    other: {
-                      'ui:title': 'Please specify your relationship',
-                      // 'ui:required': (form) => _.get('relationship.type', form) === 'other',
-                      'ui:options': {
-                        hideIf: formData => _.get('relationship.type', formData) !== 'other'
+                    'ui:title': 'Relationship to Servicemember',
+                    'ui:widget': 'radio',
+                    'ui:options': {
+                      labels: {
+                        1: 'I am the Servicemember/Veteran',
+                        2: 'Spouse or surviving spouse',
+                        3: 'Unmarried adult child',
+                        4: 'Other'
+                      },
+                      nestedContent: {
+                        1: <div className="usa-alert usa-alert-info no-background-image">You're applying as the <strong>Servicemember or Veteran</strong> whose military status and history will be used to decide if you qualify for burial in a VA national cemetery.</div>,
+                        2: <div className="usa-alert usa-alert-info no-background-image">You're applying as the <strong>legally married spouse or surviving spouse</strong> of the Servicemember or Veteran who's sponsoring this application. First, we'll ask for your information as the applicant. Then, we'll ask for your sponsor's information.</div>,
+                        3: <div className="usa-alert usa-alert-info no-background-image">You're applying as the <strong>unmarried adult child</strong> of the Servicemember or Veteran who's sponsoring this application. First, we'll ask for your information as the applicant. Then, we'll ask for your sponsor's information. You'll also need to provide supporting documents with information about your disability.</div>,
                       }
                     }
                   },
+                  /*
+                  'view:other': {
+                    'ui:title': 'Please specify',
+                    'ui:required': (form) => get(form, 'relationship.type') === '4',
+                    'ui:options': {
+                      hideIf: (form) => get(form, 'relationship.type') !== '4'
+                    }
+                  }
+                  */
```

Comment:
Doesn't map to anything in schema, so should probably be removed, but will be clarified with EOAS.